README_MACOS.md

```diff
@@ -35,87 +35,117 @@ Inside that application's folder (`SuperCollider.app/Contents/MacOS/`) are the t
-  the build instructions below, or install the current Qt version with the option `--with-qtwebkit`. As this is a non-standard install, brew will build qt5 locally (go drink a coffee).
-  If you already had Qt5, and and your build broke after an update, or if you need several Qt5 installs, you can set the version to be used by default with `brew switch`. (for example `brew switch qt5 5.5.1_2`, you can also "freeze" the Qt5 version with `brew pin`).
+      $> xcode-select --install
+
+- **homebrew** is recommended to install required libraries. See http://brew.sh for installation instructions.
+
+- **git, cmake, libsndfile, readline, and qt (version 5.5)**, installed via
+  Homebrew:
+
+      $> brew install git cmake readline
+
+  The qt-version required is outdated. To install it, the version needs to be
+  made explicit and "--forced":
+
+      $> brew install qt@5.5 --force
+
+*Note*: Recent versions of Qt do not ship with Qt5WebKit any more. It was dropped from the binary distribution since version 5.6 (functionally replaced by Qt5WebEngine). Therefore you cannot simply install the latest Qt via homebrew and rely on the defaults. You can either install qt@5.5 (accessed at /usr/local/opt/qt@5.5) and use `brew --prefix qt@5.5` in the build configuration (as shown below and recommended), or install the current Qt version with the option `--with-qtwebkit`. As this is a non-standard install, brew will build Qt locally (go drink a coffee). If you need to handle multiple Qt versions on your system, check `brew switch`. To avoid automatic update of Qt, check `brew pin`.
+
 
 Obtaining the source code
 -------------------------
 
-**Note** Please do not use non-ASCII characters (above code point 127) in your
-SuperCollider program path (i.e. the names of the folders containing SuperCollider).
-Doing so will break options to open class or method files.
+**Note** Please do not use non-ASCII characters (above code point 127) in your SuperCollider program path (i.e. the names of the folders containing SuperCollider). Doing so will break options to open class or method files.
 
 SC is on Github: https://github.com/SuperCollider/SuperCollider
 
 Get a copy of the source code:
 
-    git clone --recursive https://github.com/SuperCollider/SuperCollider.git
+    $> git clone --recursive https://github.com/SuperCollider/SuperCollider.git
+
+With `--recursive` all submodules are initialized and downloaded right away.
+
+**Note** The supercollider source embedds multiple dependencies as "submodule". Submodules are downloaded from a separate URL and need to be kept in sync with the current supercollider source checkout. Unfortunatelly this occasionally requires manual adjustments, as different SC revisions may require different sets and revisions of submodules (check `git submodule` documentation for a complete list of options).
```

Comment:
This is over-explaining (here to "build instructions"). We could provide an external link or point to `git help submodule`.